crates/rspack_plugin_javascript/src/dependency/commonjs/commonjs_export_require_dependency.rs

```diff
@@ -0,0 +1,89 @@
+use rspack_core::{
```

Comment:
A very basic implementation of export_require_dependency, full impl with optimization need parser hook

Reply:
alread create a new issue to track this